packages/terra-image/docs/README.md

```diff
@@ -18,11 +18,11 @@ import placeholderSrc from './mock/valid/path/placeholder.png';
 import invalidImageSrc from './mock/invalid/path/image.png';
```

Comment:
Would you mind adding the missing quote to the terra-image import above?

```
import Image from 'terra-image;
```

Reply:
Good catch. I fixed the react import as well.